action-server/tests/codeRunner.test.ts

```diff
@@ -0,0 +1,186 @@
+import test from "node:test";
+import assert from "assert";
+import { jsExecute } from "../src/utils/codeRunner";
+import { PoolClient } from "pg";
+import { ActionResponse } from "../src/type/types";
+
+test("Code Runner jsRunner Tests", async () => {
+  await test("capture console output", async () => {
+    const code = `
+    console.log('Hello World');
+    console.debug('Debug message');
+    console.info('Info message');
+    console.warn('Warning message');
+    console.error('Error message');
+  `;
+
+    const result: ActionResponse = await jsExecute(code, {}, {}, undefined, {} as PoolClient, 1);
+
+    assert(result.console[0].includes("[INFO] Hello World"));
+
+    // Check console outputs
+    // assert.strictEqual(result.console.log.join(), "Hello World");
+    // assert.strictEqual(result.console.debug.join(), "Debug message");
+    // assert.strictEqual(result.console.info.join(), "Info message");
+    // assert.strictEqual(result.console.warn.join(), "Warning message");
+    // assert.strictEqual(result.console.error.join(), "Error message");
```

Comment:
Should we check if the other messages are within the console array?

Reply:
Unresolving just because I added a comment to the thread while @goetzrrGit's feedback was being addressed and wanted to make sure they didn't get missed. If you feel the changes address my additions, then feel free to re-resolve